scripts/validate.sh

```diff
@@ -14,8 +14,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# This script is meant to be run locally and in CI to validate the Kubernetes
+# manifests (including Flux custom resources) before changes are merged into
+# the branch synced by Flux in-cluster.
+
+# Prerequisites
+# - yq v4.6
+# - kustomize v3.9
+# - kubeval v0.15
+
 set -o errexit
 
+echo "INFO - Downloading Flux OpenAPI schemas"
+mkdir -p /tmp/flux-crd-schemas/master-standalone-strict
+curl -sL https://github.com/fluxcd/flux2/releases/latest/download/crd-schemas.tar.gz | tar zxf - -C /tmp/flux-crd-schemas/master-standalone-strict
```

Comment:
```suggestion
curl -sL https://github.com/fluxcd/flux2/releases/latest/download/crd-schemas.tar.gz | tar zxf - -C /tmp/flux-crd-schemas/master-standalone-strict
```

This could maybe be versioned using a parameter so that older versions can be validated at a later moment as well (pipeline <> cluster ver matching).

Reply:
Not going to update this repo every time there is a new Flux release, I think it's ok like this, people that get started with this example should use the latest CLI.